apps/cyberstorm-storybook/.storybook/preview.js

```diff
@@ -0,0 +1,24 @@
+import { LinkingProvider } from "@thunderstore/components";
```

Comment:
This approach couples the "new implementation" with the "legacy implementation" in a way that requires solving when the old ones are decommissioned. It just occurred to me, that the button stories probably don't contain any links, so we should perhaps able to ditch the `LinkingProvider` completely, at least for now. Could you test if that works? I think it would be a cleaner solution since it forces the maintainer of `cyberstorm-storybook` to solve the issue if/when the stories start to use links, instead of leaving this coupling here like a ticking time bomb.

Reply:
Removed and still works.